src/app/InteractionModelEngine.h

```diff
@@ -324,12 +324,18 @@ class InteractionModelEngine : public Messaging::UnsolicitedMessageHandler,
     /**
      * @brief Function decrements the number of subscriptions to resume counter - mNumOfSubscriptionsToResume.
      *        This should be called after we have completed a re-subscribe attempt on a persisted subscription wether the attempt
-     *        was succesful or not.
+     *        was successful or not.
      */
     void DecrementNumSubscriptionsToResume();
 #endif // CHIP_CONFIG_PERSIST_SUBSCRIPTIONS
 
 #if CONFIG_BUILD_FOR_HOST_UNIT_TEST
+
+    //
+    // Get fabric table
+    //
+    FabricTable * GetFabricTable() { return mpFabricTable; }
```

Comment:
Why do we need this?  In tests, `MessagingContext` has a `GetFabricTable()` on it, no?

Reply:
just removed, yes MessagingContext has a GetFabricTable(), just update test code to use MessagingContext, thanks